src/test/mir-opt/unusual_item_types.core.ptr-drop_in_place.Vec_i32_.AddMovesForPackedDrops.before.64bit.mir

```diff
@@ -31,9 +31,9 @@ fn std::ptr::drop_in_place(_1: *mut Vec<i32>) -> () {
 
     bb6: {
         _2 = &mut (*_1);                 // scope 0 at $SRC_DIR/core/src/ptr/mod.rs:LL:COL
-        _3 = <Vec<i32> as Drop>::drop(move _2) -> [return: bb5, unwind: bb4]; // scope 0 at $SRC_DIR/core/src/ptr/mod.rs:LL:COL
+        _3 = Branch([]): for<'r> fn(&'r mut Vec::<i32>) {<Vec<i32> as Drop>::drop}(move _2) -> [return: bb5, unwind: bb4]; // scope 0 at $SRC_DIR/core/src/ptr/mod.rs:LL:COL
```

Comment:
Please reintroduce the special case for zst pretty printing

Reply:
We do special case this [here](https://github.com/rust-lang/rust/blob/70d7575bac48c3f196ce32b43c838dc42d228966/compiler/rustc_middle/src/ty/print/pretty.rs#L1353-L1354). We don't have a `ScalarPtr` for those anymore, so can't exactly mirror previous output I think.

Reply:
I mean: empty Branch should just print the path for function definition types